src/locales/en.json

```diff
@@ -20,5 +20,15 @@
     "unable.to.parse.build.gradle": "Unable to parse build.gradle: {0}.  The error is {1}",
     "error.parsing.pom": "Unable to parse pom file, the error is {0}",
     "refreshing.liberty.dashboard": "Refreshing Liberty Dashboard projects...",
-    "no.liberty.project.found.in.empty.workspace": "No Liberty project found in empty workspace"
+    "no.liberty.project.found.in.empty.workspace": "No Liberty project found in empty workspace",
+    "add.project.manually.no.projects.available.to.add": "No workspace folders available to be added to Liberty Dashboard",
+    "add.project.manually.message.0": "The selected project has been successfully added to Liberty Dashboard",
+    "add.project.manually.message.1": "The selected project already exists in Liberty Dashboard",
+    "add.project.manually.message.2": "The selected project is not a Maven or Gradle project",
```

Comment:
Is it possible to add the project name (artifact id/project name) or build file to this message? If the build file path, maybe at the end of the message.

Reply:
Added path to the message.